### PR TITLE
Infra update

### PR DESCRIPTION
**Pull request by sonicintrusion**
infrastructure/devops specific test

### CODE REVIEW

File: infra-test/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Interview
+
+This is a coding exercise to help us assess candidates looking to join the Infrastructure Engineering team at Condé Nast International.  The test is intended to be completed at home, and we expect you to spend about 2 hours on it.  If you haven't finished in that time, you can send us your progress - we don't want you to spend days on this.  We understand that your time is valuable, and appreciate any time that you contribute towards building a strong team here.  If you really cannot spare the time, you may want to take a look at Option 2.
```
Comment (olikeys) [inline infra-test/README.md]: This is rather wordy, I think it can be shortened somewhat.

Comment (sonicintrusion) [reply infra-test/README.md]: Suggestions welcome

File: infra-test/guidance.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Guidance to the candidate
+
+1. We want to make sure that you are a good fit to the culture and the work ethic of the team and company.
+1. We ask that it is returned within 3 working days of receiving the test if possible but please let us know if you require additional time. 
+1. The test is not time restricted - we do not force a maximum time limit to complete it - though it is suggested to be completed in approximately 2 hours. There is also no minimum time restriction.
+1. We want to see what you can achieve within that time that you have: how you approach the task and how you deal with the complexity of the solution. Whether you can complete the solution or not, the importance of showing your development skills is paramount.
```
Comment (olikeys) [inline infra-test/guidance.md]: that time that, should be the time that

File: infra-test/README.md (hunk 1 above)
Comment (olikeys) [reply infra-test/README.md]: # Context

This is a coding exercise to help us assess candidates looking to join the Infrastructure Engineering team at Condé Nast International, we want to discover how you handle infrastructure day-to-day.  We appreciate that your time is valuable, which is why we’ve designed this exercise to be completed in around 2 hours. If you cannot spare the time you may want to consider option 2.
 
### Things we like

- An ability to handle code like a software engineer
- Maintainability of solutions
- Resilience, particularly to Chaotic Monkeys
- Ability to develop applications locally in a consistent environments
- Concise documentation
- Containers
- Infrastructure as Code
- Monitoring, but you shouldn’t be concerned about it for now
- CI/CD, again, don’t worry about this for this exercise

**Comment by sonicintrusion** [comment]
@oli changes made

File: infra-test/guidance.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Guidance to the candidate
+
+1. We want to make sure that you are a good fit to the culture and the work ethic of the team and company.
+1. We ask that it is returned within 3 working days of receiving the test if possible but please let us know if you require additional time.
```
Comment (olikeys) [inline infra-test/guidance.md]: 3 days rather than working days?

Comment (sonicintrusion) [reply infra-test/guidance.md]: grant then the weekend if they need it

File: infra-test/guidance.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Guidance to the candidate
+
+1. We want to make sure that you are a good fit to the culture and the work ethic of the team and company.
+1. We ask that it is returned within 3 working days of receiving the test if possible but please let us know if you require additional time. 
+1. The test is not time restricted - we do not force a maximum time limit to complete it - though it is suggested to be completed in approximately 2 hours. There is also no minimum time restriction.
```
Comment (olikeys) [inline infra-test/guidance.md]: The test is not time restricted - you can spend as much, or as little, time as you like on it as long as you return it within 3 days.

Comment (sonicintrusion) [reply infra-test/guidance.md]: yes exactly.

**Pull request merged**